src/NUnitFramework/tests/Api/TestAssemblyRunnerTests.cs

```diff
@@ -223,6 +224,21 @@ public void Run_FileNotFound_ReturnsNonRunnableSuite()
                 Does.StartWith(COULD_NOT_LOAD_MSG));
         }
 
+        [Test]
+        public void RunTestsAction_WithInvalidFilterElement_ThrowsArgumentException()
+        {
+            LoadMockAssembly();
+
+            var ex = Assert.Catch(() =>
+                {
+                    var invalidFilter = TestFilter.FromXml("<filter><invalidElement>foo</invalidElement></filter>");
+                    _runner.Run(this, invalidFilter);
+                });
+
+            Assert.That(ex, Is.TypeOf<ArgumentException>());
```

Comment:
You can use `Throws<ArgumentException>` rather than `Catch` and eliminate this line.